controlpanel/frontend/jinja2/base.html

```diff
@@ -107,16 +108,16 @@
         "active": page_name == "webapps"
       },
       {
-        "hide": not request.user.is_superuser,
```

Comment:
FYI: I'll focus my review on this line as the rest seem to be carried from a different/related branch.

I'm keen on having the permissions' source of true in few key places in the codebase, one of them currently being [the `api/rules.py` file](https://github.com/ministryofjustice/analytics-platform-control-panel/blob/master/controlpanel/api/rules.py#L196-L200).

This has some benefits, including predictability (assuming someone knows permissions are defined there) and also the fact that reduces duplication and if for whatever reason we make the current logic to show the parameters more complicated we only need to change it there.

The `api.list_parameter` is already defined so that would mean use it in this template:

```jinja
"hide": not request.user.has_perm("api.list_parameters"),
```

(👆 or something like that, not tested)

Mind you, the net result is the same and you may argue it's annoying to have to jump in the `api/rules.py` file to check who can list parameters but it reduces the chance of future bugs or different part of the CP showing different things in the event of changing this simple logic to list parameters.

My 2 cents anyway ☺️

Reply:
Aha... I'll revise. I completely agree WRT single source of truth for permissions and I'll re-use the thing you pointed out. :+1:

Reply:
OK... it was interesting to learn about the rules and how they're applied in templates. But, AFAICT, we want *all* users to be able to set parameters. As a result, we don't need any `hide` flag on the tab.